src/everest/bin/everest_script.py

```diff
@@ -100,22 +97,13 @@ def run_everest(options):
             job_name = fm_job.split()[0]
             logger.info("Everest forward model contains job {}".format(job_name))
 
-        with PluginSiteConfigEnv():
-            ert_config = ErtConfig.with_plugins().from_dict(
-                config_dict=generate_everserver_ert_config(
-                    options.config, options.debug
-                )
-            )
-
         makedirs_if_needed(options.config.output_dir, roll_if_exists=True)
-
-        with open_storage(ert_config.ens_path, "w") as storage, PluginSiteConfigEnv():
-            context = start_server(options.config, ert_config, storage)
-            print("Waiting for server ...")
-            wait_for_server(options.config, timeout=600, context=context)
-            print("Everest server found!")
-            run_detached_monitor(options.config, show_all_jobs=options.show_all_jobs)
-            wait_for_context()
+        server_config = generate_everserver_config(options.config)
+        await start_server(options.config, server_config, options.debug)
+        print("Waiting for server ...")
+        wait_for_server(options.config, timeout=600)
```

Comment:
Do/should we need this call when `start_server` is async?

Reply:
Yes, though they might be merged, but not sure they should. We are awaiting the submission to the queue, while we are waiting for the server to start afterwards, which takes some time, and there is no report from the driver when that happens, as it does not know. It just knows that it is running.